compiler/z/codegen/BinaryEvaluator.cpp

```diff
@@ -1246,12 +1246,12 @@ lDivRemGenericEvaluator64(TR::Node * node, TR::CodeGenerator * cg, bool isDivisi
                TR::Register * tempRegister2 = cg->allocate64bitRegister();
                generateRSInstruction(cg, TR::InstOpCode::SRAG, node, tempRegister2, firstRegister, 63);
                generateRRInstruction(cg, TR::InstOpCode::LGR, node, tempRegister1, firstRegister);
-               generateRILInstruction(cg, TR::InstOpCode::NIHF, node, tempRegister2, (int32_t)((absValueOfDenomintor-1)>>32) );
-               generateRILInstruction(cg, TR::InstOpCode::NILF, node, tempRegister2, (int32_t)(absValueOfDenomintor-1));
+               generateRILInstruction(cg, TR::InstOpCode::NIHF, node, tempRegister2, static_cast<int32_t>((absValueOfDenomintor-1)>>32) );
```

Comment:
Since this PR touches most of the references already, could you fix the spelling of "Denominator" in the `absValueOfDenomintor` local variable?